Vagrantfile

```diff
@@ -63,8 +63,12 @@ Vagrant.configure('2') do |config|
     config.hostmanager.enabled = true
     config.hostmanager.manage_host = true
     config.hostmanager.aliases = hostnames + redirects
+  elsif Vagrant.has_plugin? 'landrush'
```

Comment:
This probably isn't going to work as expected, many users will have both hostmanager and landrush installed.